src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs

```diff
@@ -128,20 +128,43 @@ public static IResourceBuilder<GarnetResource> WithDataBindMount(this IResourceB
     /// <code lang="csharp">
     /// var cache = builder.AddGarnet("cache")
     ///                    .WithDataVolume()
-    ///                    .WithPersistence(TimeSpan.FromSeconds(10), 5);
+    ///                    .WithPersistence(TimeSpan.FromSeconds(10));
     /// </code>
     /// </example>
     /// <param name="builder">The resource builder.</param>
     /// <param name="interval">The interval between snapshot exports. Defaults to 60 seconds.</param>
     /// <param name="keysChangedThreshold">The number of key change operations required to trigger a snapshot at the interval. Defaults to 1.</param>
     /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    [Obsolete("This method is obsolete and will be removed in a future version. Use the overload without the keysChangedThreshold parameter.")]
```

Comment:
Could you please help me understand why we are making this obsolete? It seems that it doesn't work due to a regression, and should be fixed a newer version?

Reply:
Because of `keysChangedThreshold`.
Garnet does not have this arg.